kuma/static/styles/components/wiki/indicators.scss

```diff
@@ -206,14 +269,9 @@ icons for both types of indicators
     @include indicator-icon('\f017'); /* clock */
 }
 
+.esHarmony,
 .experimental,
-.experimentalBadge,
-/* es7 and harmony macros  */
-.overheadIndicator[style='background: #9CF49C;'],
-/*
-IE 11 converts hex to rgb, so target that as well.
-*/
-.overheadIndicator[style='background: rgb(156, 244, 156);'] {
+.experimentalBadge {
```

Comment:
Same comment here as above, unless this class is already widely used in this form. If that is the case, we can clean this up later.

Reply:
`.experimentalBadge` is, like most other legacy classes in this file, used quite a bit in this form on MDN. 😔

Reply:
No worries, we can clean this up later.